core-bundle/src/Routing/ResponseContext/WebpageResponseContext.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Routing\ResponseContext;
+
+class WebpageResponseContext extends ResponseContext
+{
+    /**
+     * @var string
+     */
+    private $title = '';
+
+    /**
+     * @var string
+     */
+    private $description = '';
+    /**
+     * @var string
+     */
+    private $robotsMetaTagContent = 'index,follow';
+
+    public function getTitle(): string
+    {
+        return $this->title;
+    }
+
+    public function setTitle(string $title): self
+    {
+        $this->title = $title;
+
+        return $this;
+    }
+
+    public function getDescription(): string
+    {
+        return $this->description;
+    }
+
+    public function setDescription(string $description): self
+    {
+        $this->description = $description;
+
+        return $this;
+    }
+
+    public function getRobotsMetaTagContent(): string
+    {
+        return $this->robotsMetaTagContent;
+    }
+
+    public function setRobotsMetaTagContent(string $robotsMetaTagContent): self
+    {
+        $this->robotsMetaTagContent = $robotsMetaTagContent;
+
+        return $this;
+    }
```

Comment:
Not sure if that got lost: I think we should have a generic meta attribute parameter bag which would allow us to add arbitrary meta tags. This would also support @m-vo's use case.

Reply:
It did not. I just said this was not part of this PR and resolved the conversation.
There's a lot more stuff that needs to go in the context like scripts etc. but again, not part of this PR.

Reply:
then we shouldn't add the meta robots tag neither, should we?

Reply:
Why not? It works the same as description.

Reply:
See my latest comment in the PR about this. There's quite a few places in our code where we want to override the robots meta as well as the description so I think they deserve the shortcut methods we currently have. We can map them nicely onto a future whatever-head-tag-bag. Marking this conversation as resolved as well.